extension/chrome/content/download.js

```diff
@@ -36,7 +36,9 @@
 
 var Cc = Components.classes;
 var Ci = Components.interfaces;
+var Cu = Components.utils;
 
+Cu.import("resource://gre/modules/Downloads.jsm");
```

Comment:
By using this module we have to bump the minversion of Firefox/Gecko to 26.0 in the manifest file.